spec/classes/rabbitmq_spec.rb

```diff
@@ -6,7 +6,7 @@
     let(:facts) {{ :osfamily => 'Unsupported' }}
 
     it 'we fail' do
-      expect { subject }.to raise_error(/not supported on an Unsupported/)
+      expect { is_expected.to compile }.to raise_error(RSpec::Expectations::ExpectationNotMetError, /not supported on an Unsupported/)
```

Comment:
Hi @cmurphy tested this morning with matching the regex and not the error type and that was ok. I think this would be neater.